pages/documentation/developer/repositories/index.md

```diff
@@ -2,215 +2,214 @@
 This [repository](https://github.com/powsybl/powsybl-entsoe) provides components specific to ENTSO-E-orientated processes.
 
-**Reviewers:** [miovd](https://github.com/miovd), [annetill](https://github.com/annetill), [colinepiloquet](https://github.com/colinepiloquet)    
-**Committers:** [miovd](https://github.com/miovd), [colinepiloquet](https://github.com/colinepiloquet)  
+**Reviewers:** [phiedw](https://github.com/phiedw), [annetill](https://github.com/annetill), [colinepiloquet](https://github.com/colinepiloquet)     
+**Committers:** [phiedw](https://github.com/phiedw)   
 
 ### [powsybl-open-rao](powsybl-open-rao.md) [![GitHub release](https://img.shields.io/github/release/powsybl/powsybl-open-rao.svg?sort=semver)](https://github.com/powsybl/powsybl-open-rao/releases/)
 This [repository](https://github.com/powsybl/powsybl-open-rao) provides a modular engine for remedial actions optimization.
 
-**Reviewers:** [bqth29](https://github.com/bqth29), [benrejebmoh](https://github.com/benrejebmoh), [Godelaine](https://github.com/Godelaine), [jipea](https://github.com/jipea), [MartinBelthle](https://github.com/MartinBelthle), [murgeyseb](https://github.com/murgeyseb), [pet-mit](https://github.com/pet-mit), [phiedw](https://github.com/phiedw)
-**Committers:** [bqth29](https://github.com/bqth29), [benrejebmoh](https://github.com/benrejebmoh), [Godelaine](https://github.com/Godelaine), [jipea](https://github.com/jipea), [MartinBelthle](https://github.com/MartinBelthle), [murgeyseb](https://github.com/murgeyseb), [pet-mit](https://github.com/pet-mit), [phiedw](https://github.com/phiedw)
+**Reviewers:** [bqth29](https://github.com/bqth29), [benrejebmoh](https://github.com/benrejebmoh), [Godelaine](https://github.com/Godelaine), [jipea](https://github.com/jipea), [MartinBelthle](https://github.com/MartinBelthle), [murgeyseb](https://github.com/murgeyseb), [pet-mit](https://github.com/pet-mit), [phiedw](https://github.com/phiedw)  
+**Committers:** [bqth29](https://github.com/bqth29), [benrejebmoh](https://github.com/benrejebmoh), [Godelaine](https://github.com/Godelaine), [jipea](https://github.com/jipea), [MartinBelthle](https://github.com/MartinBelthle), [murgeyseb](https://github.com/murgeyseb), [pet-mit](https://github.com/pet-mit), [phiedw](https://github.com/phiedw)  
 
-### [powsybl-metrix](powsybl-metrix.md) [![GitHub release](https://img.shields.io/github/release/powsybl/powsybl-metrix.svg?sort=semver)](https://github.com/powsybl/powsybl-metrix/releases/)
-This [repository](https://github.com/powsybl/powsybl-metrix) provides modules to run optimal power load flow on several network variants. Variants are generated through time series mapping on a base case.
+### [powsybl-dependencies](//TODO) [![GitHub release](https://img.shields.io/github/release/powsybl/powsybl-dependencies.svg?sort=semver)](https://github.com/powsybl/powsybl-dependencies/releases/)
```

Comment:
Add a dedicated page or remove the link on the title (instead of referencing the "todo" website).